content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -123,6 +123,30 @@ const prisma = new PrismaClient({
 })
 ```
 
+### <inlinecode>adapter</inlinecode>
+
+Defines an instance of a driver adapter. See also [Drivers](/concepts/components/drivers) <span class="concept"></span>.
+
+This is available from version 5.4.0 and upwards.
```

Comment:
This probably should mention that this needs the `driverAdapters` preview feature right now.